WordPress/Jetpack/Classes/ViewRelated/WordPress-to-Jetpack Migration/Success card/MigrationSuccessCardView.swift

```diff
@@ -106,7 +106,7 @@ extension MigrationSuccessCardView {
             return false
         }
 
-        let isFeatureFlagEnabled = FeatureFlag.contentMigration.enabled
+        let isFeatureFlagEnabled = true
```

Comment:
This property is now redundant.

Reply:
Fixed! [8068d3a](https://github.com/wordpress-mobile/WordPress-iOS/pull/21499/commits/8068d3a809e4ada5ea4fcc992a4229bd63555c71)